### ISSUE TITLE
Discounted shipping label pop-up blocking access to edit order

### ISSUE THREAD

**Issue by AashikP** [Automattic/woocommerce-services#2128]
*Customer Reports*

3168571-zen
3204865-zen
23345765-hc
3168571-zen

**Steps to replicate**

1. Activate WooCommerce Services on a store where it wasn't activated before
2. Attempt to open an order (Chrome) to find that you're not able to click anything on the screen:
    ![https://d.pr/i/H9SxWc+](https://d.pr/i/H9SxWc+)

It does work on some browsers for some users. For example, this is what I see on Firefox:

![https://d.pr/i/rnHFbP+](https://d.pr/i/rnHFbP+)

Dismissing the notice on firefox while logged in as the same user will ensure that the issue is fixed the next time I access the order edit page in Chrome.

Here are my browser details:

Chrome (bug): https://www.whatsmybrowser.org/b/41D1T
Firefox (works fine): https://www.whatsmybrowser.org/b/HI6DL

However, user in 3168571-zen reported that none of the browsers worked for them:

https://www.whatsmybrowser.org/b/ZK5RL
https://www.whatsmybrowser.org/b/5SEC9
https://www.whatsmybrowser.org/b/V6Y18

**Expected Behavior**

Either fix the problem with pop-up hiding other options on some browsers or make it a notification like other WordPress notifications and not a pop-up.

Marking as high priority because without reaching out to support or performing a complete conflict test, most users wouldn't know what exactly is happening.

Previous slack discussion: p1593417215220200-slack-fulfill-support

**Edit:** In my case, I could not replicate the behavior on my Macbook. The problem could only be replicated on iMac.

**Comment by selectedselections** [comment]
reported in 3204865-zen as well 

screencast of issue:  https://www.loom.com/share/f7dd79eba3fa48d495105ffde8b240dc

**Comment by RiaanKnoetze** [comment]
From the original bug report: 

> However, user in 3168571-zen reported that none of the browsers worked for them

User in 3204865-zen also reports that neither Firefox nor Chrome is working for them and shared the following screencast: https://www.loom.com/share/d52b86e11f124ff080dc9876c8b68a67

Considering that this breaks order management entirely, I'm bumping to critical.

**Comment by harriswong** [comment]
Hi, 

I took a look at this and I could not reproduce the problem on both local environment and live environment, against both Chrome (latest 84) and FIrefox (latest 79).

I can see both showing:
![image](https://user-images.githubusercontent.com/572862/90051521-06c47e80-dc95-11ea-8411-a055e74fc39c.png)

I am wondering if these are caching issues on their host providers. Were you able to reproduce this on a fresh installation?

**Comment by AashikP** [comment]
Thank you for checking, @harriswong 

> Were you able to reproduce this on a fresh installation?

Yep. The screenshots on the original report are from my own test sites. Heads up that it could also be due to the screen size (updating original report). 

While I did have the problem in iMac, that wasn't the case using a laptop.

**Comment by KristinaKay** [comment]
Just had a instance of this in 23345765-hc

**Comment by frosso** [comment]
Thanks for the reports! I believe I have a fix ready in https://github.com/Automattic/woocommerce-services/pull/2139 - feel free to check out the changes there

**Comment by AashikP** [comment]
Thank you for the PR @frosso.

I downloaded the PR as a zip file, but I'm still seeing the same behavior on my own test site. Here's the file I downloaded https://d.pr/f/g966yj

**Comment by frosso** [comment]
@AashikP thanks for testing it out! In order to check out the changes, you'll need a compiled version of the plugin. I created one for you: https://d.pr/f/x3YDhN

Let me know if this works better!

**Comment by AashikP** [comment]
> you'll need a compiled version of the plugin.

Thank you for reminding me @frosso  🤦🏻‍♂️

While the compiled version did not work initially, I _could get it to work while logged in via incognito_. Probably caching. So yeah, the fix indeed seems to work :)

**Comment by frosso** [comment]
@AashikP Thanks again for taking the time to test it!!

**Comment by madeincosmos** [comment]
Also reported in 3237565-zen

Here the problem only happened when Shipping Labels were disabled under WooCommerce > Settings > Shipping > WooCommerce Services. Enabling them back works as a temporary workaround:

![Screenshot](https://d.pr/i/AW5L5n+)
Screenshot: https://d.pr/i/AW5L5n

**Comment by jessepearson** [comment]
Got another one in 3168571-zen, and added it and the rest to the top of the report. 

I tried to update using the compiled version above, but it didn't install, most likely because it's an AT site on WP.com.

![](https://d.pr/i/EeaPBG+)
Image Link: https://d.pr/i/EeaPBG

@madeincosmos replied as I was replying, so I went off of what she said and toggled the Shipping Labels off and on again and that seems to have corrected the issue for now.
